models/engine/db_storage.py

```
@@ -97,3 +97,30 @@ def close(self):
         close method on the class Session
         '''
         self.__session.close()
+
+    def get(self, cls, id):
+        '''
+            returns object by class and id
+        '''
+        result = self.__session.query(eval(cls)).all()
+        for obj in result:
+            if obj.id == id:
+                return obj
+
+    def count(self, cls=None):
```

"Return total..."
Maybe raise a specific exception instead of a general one?